config/fxdata/objects.cfg

```diff
@@ -713,3 +713,13 @@ Properties = CHOWNED_ON_ROOM_CLAIM
 Name = SPELLBOOK_POSS
 Genre = SPELLBOOK
 Properties = CHOWNED_ON_ROOM_CLAIM
+
+[object136]
+Name = NUM_1
+Genre = DECORATION
+Properties = CHOWNED_ON_ROOM_CLAIM DESTROYED_ON_ROOM_PLACE
+ObjectSize = 200
+Image3d = "mbox.tga"
+Region3d = "0,0,58,0"
+Image = "mbox.tga"
```

Comment:
If I edit the mbox.tga with paint.net, even if I don't use any new colors, the image no longer shows up in FX. Do you have a good suggestion on how to create .tga files for free in a way that FX can read them?

Reply:
One may use free as speech and beer Gimp 
Just add .GPL file

Reply:
[keeper.gpl.zip](https://github.com/dkfans/keeperfx/files/6357816/keeper.gpl.zip)

Reply:
Set to indexed
![image](https://user-images.githubusercontent.com/68837817/115706441-5146e080-a376-11eb-82a8-a7f45634d77d.png)

Reply:
![image](https://user-images.githubusercontent.com/68837817/115706535-6f144580-a376-11eb-8e87-449d48db76d2.png)

Reply:
Set palette

Reply:
Layer > Transparency > Remove Alpha

Reply:
![image](https://user-images.githubusercontent.com/68837817/115706849-c0243980-a376-11eb-9c95-3892d31f1311.png)

Reply:
File > Export As > good.tga

Reply:
Correct options like that:
![image](https://user-images.githubusercontent.com/68837817/115707034-f82b7c80-a376-11eb-98d7-4a6b4d40b5fd.png)

Reply:
And here is a working tga
[good.tga.zip](https://github.com/dkfans/keeperfx/files/6357864/good.tga.zip)

Reply:
Yes, with GIMP and your instructions I managed to get the image in.

Reply:
This is now obsolete